css/selectors/autofill.json

```diff
@@ -0,0 +1,66 @@
+{
+  "css": {
+    "selectors": {
+      "autofill": {
+        "__compat": {
+          "description": "<code>:autofill</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:autofill",
+          "support": {
+            "chrome": {
+              "version_added": "1",
+              "prefix": "-webkit-"
+            },
+            "chrome_android": {
+              "version_added": "18",
+              "prefix": "-webkit-"
+            },
+            "edge": {
+              "version_added": "79",
+              "prefix": "-webkit-"
+            },
+            "firefox": {
+              "version_added": "86",
+              "notes": "Firefox has also implemented -webkit-autofill as an alias."
```

Comment:
This note should be reported as two support statements in an array, one with a prefix and one without.